docs/deployment_policy.md

```diff
@@ -0,0 +1,99 @@
+      - `priority_value`: The priority value assigned to this version. Priority is expressed in human terms, where a lower ordinal value means higher priority. Priority values within the list are not required to be sequential, just unique within the list. When deploying a service, OpenHorizon will attempt to deploy the higest priority version first. If the service is not successfully started, the next highest version will be attempted.
+      - `retries`: The number of times to retry starting a failed service.
+      - `retry_durations`: The number of seconds (i.e. elapsed time) in which the indicated number of `retries` must occur before giving up and moving on to the next highest priority service version.
+  - `nodeHealth`: For nodes that are expected to remain network connected to the management, these setting indicate how agressive the Agbot should be in determining if a node is out of policy.
+    - `missing_heartbeat_interval`: The number of seconds a heartbeat can be missed (from the perspective of the management hub) until the node is considered missing. When a node is detected as missing, its agreements are cancelled by the Agbot.
+    - `check_agreement_status`: The number of seconds between checks (by the management hub) to verify that the node still has an agreement for this service.
+- `properties`: Policy properties as described [here](./properties_and_constraints.md) which a node policy constraint can refer to.
+- `constraints`: Policy constraints as described [here](./properties_and_constraints.md) which refer to node policy properties.
+- `userInput`: This section is used to set service variables for any service (including this service) that is deployed as a result of deploying this service.
+  - `serviceUrl`: The name of the service to be configured. This is the same value as found in the `url` field [here](./service_def.md).
+  - `serviceOrgid`: The organization in which the service in `serviceUrl` is defined.
+  - `serviceArch`: The hardware architecture of the service in `serviceUrl`, or `*` to indicate any compatible architecture. This is the same value as found in the `arch` field [here](./service_def.md).
+  - `serviceVersionRange`: A version range indicating the set of service versions to which this variable setting should be applied.
+  - `inputs`: A list of service variables to set.
+    - `name`: The name of the variable. This is the same as a variable name found in `userInputs` as defined [here](./service_def.md).
+    - `value`: The value to be assigned to the variable. Service variables are typed as described in `userInputs` defined [here](./service_def.md).
+
+The following is an example of a deployment policy that deploys a service called `my.company.com.service.this-service`.
+The service is defined within organization `yourOrg`.
+This policy will deploy the service to any node which matches one of the architectures for which the service is defined.
```

Comment:
This statement need to be modified because the policy has constraints. ["aNodeProperty==someValue"].